src/Microsoft.AspNetCore.Routing/Template/RouteTemplate.cs

```diff
@@ -59,5 +59,24 @@ private string DebuggerToString()
         {
             return string.Join(SeparatorString, Segments.Select(s => s.DebuggerToString()));
         }
+
+        /// <summary>
+        /// Gets the parameter matching the given name 
+        /// </summary>
+        /// <param name="name"></param>
+        /// <returns>matching parameter or null if none matches</returns>
```

Comment:
`The matching parameter or <c>null></name> if no parameter matches the given name.`